core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/release/BitbucketCloudReleaser.java

```diff
@@ -0,0 +1,283 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright 2020-2023 The JReleaser authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jreleaser.model.internal.release;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.jreleaser.model.Active;
+import org.jreleaser.model.api.common.CommitAuthor;
+import org.jreleaser.model.api.release.Changelog;
+
+import java.util.Map;
+
+import static java.util.Collections.unmodifiableMap;
+
+/**
+ * @author Hasnae Rehioui
+ * @since 1.6.0
+ */
+public final class BitbucketCloudReleaser extends BaseReleaser<org.jreleaser.model.api.release.BitbucketCloudReleaser, BitbucketCloudReleaser> {
+
+    // TODO: viqueen - can we use immutables library ?
+    //  is this code generated by any chance ? it feels like it can be
+    @JsonIgnore
+    private final org.jreleaser.model.api.release.BitbucketCloudReleaser immutable = new org.jreleaser.model.api.release.BitbucketCloudReleaser() {
```

Comment:
re: immutables library. One reason to follow this pattern is that the resulting class is anonymous. The `immutable` variable is passed to JReleaser extensions as read-only views of the model. The idea is that the use of an anonymous class would prevent consumers from prodding into the internals, or at least make it more difficult.